tools/XCConfigDumper/Sources/XCConfigDumper/Record.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Line.swift
+//  XCConfigDumper
+//
+//  Created by Vlad Solomenchuk on 8/9/19.
+//
+
+import Foundation
+
+private func tokenize(text: String) -> [String] {
```

Comment:
I'd suggest using the program mentioned in the PR for this https://github.com/jerrymarino/XcodeCompilationDatabase/blob/master/Sources/XcodeCompilationDatabaseCore/XcodeCompilationDatabase.swift#L8 which implemented Xcode log parsing

Reply:
Hi @jerrymarino, thanks for the suggestion, but the library you've mentioned doesn't really work for this task. This PR is very simple and covered with tests as well